src/cosalib/qemuvariants.py

```diff
@@ -26,6 +26,9 @@
 # BASEARCH is the current machine architecture
 BASEARCH = get_basearch()
 
+# Default flags for the creation of tarfiles
+DEFAULT_TAR_FLAGS = '-Schgh'
```

Comment:
Expanding the comment with *why* we are picking these flags would be useful.

We always want `-c`, so it seems better to provide that automatically in the 

Minor: `h` is there twice.